finance/contracts/src/test/kotlin/net/corda/finance/contracts/asset/ObligationTests.kt

```diff
@@ -549,7 +547,18 @@ class ObligationTests {
                 this.verifies()
             }
         }
-
+        ledgerServices.ledger(DUMMY_NOTARY) {
```

Comment:
We need a unit test to test when issuerRef is different between acceptedIssueProduct and cash's issuerRef, the test can still pass

Reply:
Done